docs/releasing.md

```diff
@@ -2,41 +2,82 @@
+   git checkout main
+   git pull --tags
+   ```
 
-2. Run `yarn release` which uses [`standard-version`](https://github.com/conventional-changelog/standard-version) to:
+2. Determine the version number of the new release
+   
+   ```
+   yarn release --dry-run
+   ```
+   You should see output similar to `✔ bumping version in package.json from {old.version.number} to {new.version.number}`. Take note of the `{new.version.number}` which will be used throughout the remainder of release process.
+   
+   > :information_source: `yarn release` uses [`standard-version`](https://github.com/conventional-changelog/standard-version) which, combined with adherence to [conventional commits](https://www.conventionalcommits.org/) enforced on main via title linting and using `squash and merge`.
+   [`--dry-run`](https://github.com/conventional-changelog/standard-version#dry-run-mode) logs all of the output and lets you know what `standard-version` *would* do (without actually making changes) so that you can verify that everything aligns with your expectations. 
+   Most importantly, the log will explain how the version number would be bumped, based on new commits to `main`. 
 
-   - Determine the new version based on new commit messages
-   - Generate a new entry in the changelog with the version, release notes, and today's date.
-   - Commit all of the above changes with the message `chore(release): <version>`
+3. Create a release branch for the new version to be released, where `{new.version.number}` might be `2.4.1`, for example.
+   
+   ```
+   git checkout -b release-{new.version.number} 
+   ```
 
-3. Push changes to a new branch following the naming pattern: `release-<version>`
+4. On the newly created release branch, allow `standard-version` to create the release
+   
+   ```
+   yarn release
+   ```
+   
+   Without using `--dry-run`, `standard-version` will commit the following changes with the message `chore(release): {new.version.number}`:
+   - Bump the version in `package.json`
+   - Generate a new entry in `CHANGELOG.md` with the version, release notes, and today's date.
 
-   - For example: `git checkout -b release-1.1.0`
+5. Push the changes to Github
+   
+   ```
+   git push -u origin release-{new.version.number}
+   ```
 
-4. Open a PR for the release branch against `main`, with the changelog generated by the previous step included in the PR description.
+6. Open a PR for the release branch against `main`
 
    - PR title should be `chore(release): <version>`
+   - Paste the the changelog generated by `standard-version` in the PR description/body.
    - Ask for approvals from stakeholders, perform testing on applications, etc.
+      - It can be helpful to reviewers to provide a diff of all the files that have changed since the last release. To do this, add a compare link as a comment to the PR, or in the description/body:
+      `https://github.com/trussworks/react-uswds/compare/{previous.release.tag}...release-{new.version.number}`
```

Comment:
⭐ yup. This is something I would love to automate is commenting this on PRs with the release tag. I thought I made an issue for this but I don't see it so made https://github.com/trussworks/react-uswds/issues/2054 in case automation is something to pursue here.

Reply:
Nice! Yea I remember that issue as well, buuuuut I have no idea where it is. Thanks for filing a new one!